### PR TITLE
New version: BleachBit.BleachBit version 4.6.0

### PR DESCRIPTION
**Pull request by Exorcism0666**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/132845)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification.
>[!important]
>Please recheck these Pull Request carefully before merging. If it is a incorrect-removing Pull Request and the URLs don't have any issue while manual checking, please close this Pull Request directly(It's best to let Exorcism0666 know).

良い一日をお過ごしください！

**Comment by stephengillie** [comment]
Hi @Exorcism, 

Automatic Validation ended with:
>  Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 
'manifests\b\BleachBit\BleachBit\4.6.0', expected 
'manifests\b\BleachBit\BleachBit\4.6.0.2537' 

The PackageVersion has an extra `.2537` that's missing from the file path. They have to match.

**Comment by Exorcism0666** [comment]
image

File: manifests/b/BleachBit/BleachBit/4.6.0.2537/BleachBit.BleachBit.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: BleachBit.BleachBit
+PackageVersion: 4.6.0.2537
+InstallerType: nullsoft
+Installers:
+- Architecture: neutral
```
Comment (OfficialEsco) [inline manifests/b/BleachBit/BleachBit/4.6.0.2537/BleachBit.BleachBit.installer.yaml:8]:
```suggestion
- Architecture: x64
```

File: manifests/b/BleachBit/BleachBit/4.6.0.2537/BleachBit.BleachBit.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+InstallerType: nullsoft
+Installers:
+- Architecture: neutral
+  Scope: user
+  InstallerUrl: https://download.bleachbit.org/BleachBit-4.6.0-setup.exe
+  InstallerSha256: 4be77c019328203bc16fcd6ab62b41e26e14edfdc4572448d0ddb101ba796ec9
+  InstallerSwitches:
+    Silent: /S
+    Custom: /currentuser
+- Architecture: neutral
```
Comment (OfficialEsco) [inline manifests/b/BleachBit/BleachBit/4.6.0.2537/BleachBit.BleachBit.installer.yaml:15]:
```suggestion
- Architecture: x64
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Bet this is the issue

**Review by OfficialEsco — APPROVED** [review]

**Comment by OfficialEsco** [comment]
Just checked the Artifact from the installation
https://dev.azure.com/ms/winget-pkgs/_build/results?buildId=528918&view=artifacts&pathAsName=false&type=publishedArtifacts
It complains about msvcr100.dll not found. which is "Microsoft Visual C++ 2010 Redistributable"
InstallationVerification-x64-machine-Locale_Undefined-ErrorScreenshot_20240107135849

File: manifests/b/BleachBit/BleachBit/4.6.0.2537/BleachBit.BleachBit.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: BleachBit.BleachBit
+PackageVersion: 4.6.0.2537
+InstallerType: nullsoft
+Installers:
+- Architecture: x64
```
Comment (OfficialEsco) [inline manifests/b/BleachBit/BleachBit/4.6.0.2537/BleachBit.BleachBit.installer.yaml:8]:
```suggestion
- Architecture: x86
```

File: manifests/b/BleachBit/BleachBit/4.6.0.2537/BleachBit.BleachBit.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+InstallerType: nullsoft
+Installers:
+- Architecture: x64
+  Scope: user
+  InstallerUrl: https://download.bleachbit.org/BleachBit-4.6.0-setup.exe
+  InstallerSha256: 4be77c019328203bc16fcd6ab62b41e26e14edfdc4572448d0ddb101ba796ec9
+  InstallerSwitches:
+    Silent: /S
+    Custom: /currentuser
+- Architecture: x64
```
Comment (OfficialEsco) [inline manifests/b/BleachBit/BleachBit/4.6.0.2537/BleachBit.BleachBit.installer.yaml:15]:
```suggestion
- Architecture: x86
```

File: manifests/b/BleachBit/BleachBit/4.6.0.2537/BleachBit.BleachBit.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,23 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: BleachBit.BleachBit
+PackageVersion: 4.6.0.2537
+InstallerType: nullsoft
```
Comment (OfficialEsco) [inline manifests/b/BleachBit/BleachBit/4.6.0.2537/BleachBit.BleachBit.installer.yaml:6]:
```suggestion
InstallerType: nullsoft
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2010.x86
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
NOW it should work without any issues

**Review by OfficialEsco — APPROVED** [review]

**Comment by Exorcism0666** [comment]
@OfficialEsco always not LMAO

**Comment by OfficialEsco** [comment]
errr its another package with

> Installation failed with exit code -1978334972

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Installation failed with exit code -1978334972

(Automated response - build 530.)

**Pull request merged**